DQMServices/Components/plugins/MEtoEDMConverter.h

```diff
@@ -51,8 +51,10 @@
 #include "TObjString.h"
 
 class MEtoEDMConverter : public edm::one::EDProducer<edm::one::WatchRuns,
+                                                     edm::one::WatchLuminosityBlocks,
                                                      edm::EndLuminosityBlockProducer,
-                                                     edm::EndRunProducer>
+                                                     edm::EndRunProducer,
+                                                     edm::one::SharedResources>
```

Comment:
what is the purpose of the `SharedResources` ?

Reply:
I also do not understand its usage.

Reply:
It gives the `usesResource` call, which is a quick way to get a global lock for legacy stuff. Exactly what legacy DQM needs.

Though ideally, the final version of this PR will simply make that all thread save and we can get rid of `usesResource("DQMStore")` everywhere.